src/wallet/walletdb.cpp

```diff
@@ -790,17 +793,19 @@ DBErrors CWalletDB::LoadWallet(CWallet* pwallet)
             string strType, strErr;
             if (!ReadKeyValue(pwallet, ssKey, ssValue, wss, strType, strErr))
             {
-                // losing keys is considered a catastrophic error, anything else
-                // we assume the user can live with:
-                if (IsKeyType(strType))
+                if (strType == "networkinfo") {
```

Comment:
It's a bit unfortunate that the key types have to be known both here and in ReadKeyValue, but since that was already the case with the `"tx"` key this seems like an okay solution.